### DISCUSSION TITLE
python and other errors with Forge

### DISCUSSION THREAD

**Discussion by Weatherby43** [lllyasviel/stable-diffusion-webui-forge#766] [Q&A]
Hi all,

I have 2 instances of Forge running on 2 different boxes. One has a 3060 the other a 3090. The 3090 install gives me fits. Installing Forge, I have to run the webui-user.bat more than once to get it to complete installing all of it's parts (on a venv delete and reinstall). once it installs, I get frequent 'quits' where I'll get errors on the gradio gui and when I check the console, forge has just stopped and gone to 'hit any key to continue' where, if I hit it, it dumps to command line. There are no errors, nothing. It just stops. it does this a lot. The 3060 box, does not. Both have git pull in the webui-user.bat so pull updates as they happen. So they're both the same version etc. 

I've tried 3.10.6-3.10.11 of python, all still give the same errors. 

Now, on the 3090 box, I get errors on other python things like supir as well. Supir I have to run the damn bat a dozen times to get it to start once. Also, I noticed when it was downloading models that it kept stopping the download and erroring with something is missing attribute length error. Also, installing things with pip is a nightmare of errors, try again, oh look it installed one more thing, oh run again..etc... 

Anyone have any idea what might be going on here?

**Comment by SolemAubrey** [comment]
I am having no success installing Forge at all. I keep getting something about Cannot Import Undefined from Pydantic, even after upgrading pip. I don't know what it wants me to do.
But A1111 worked fine for me before.
